startup_scripts/180_vrfs.py

```diff
@@ -0,0 +1,46 @@
+from ipam.models import VRF
+from tenancy.models import Tenant
+from ruamel.yaml import YAML
+from extras.models import CustomField, CustomFieldValue
+
+from pathlib import Path
+import sys
+
+file = Path('/opt/netbox/initializers/vrfs.yml')
+if not file.is_file():
+  sys.exit()
+
+with file.open('r') as stream:
+  yaml = YAML(typ='safe')
+  vrfs = yaml.load(stream)
+
+  optional_assocs = {
+    'tenant': (Tenant, 'name')
```

Comment:
I think you're missing the association for the `TenantGroup` here:

```suggestion
    'tenant': (Tenant, 'name')
    'tenant_group': (TenantGroup, 'name')
```

Reply:
I'm not sure if VRF is associated with a TenantGroup directly, I think only with Tenant. https://github.com/netbox-community/netbox/blob/develop/netbox/ipam/models.py